src/library-authoring/LibraryAuthoringPage.tsx

```diff
@@ -268,6 +255,14 @@ const LibraryAuthoringPage = ({ returnToLibrarySelection }: LibraryAuthoringPage
               <Tab eventKey={ContentType.components} title={intl.formatMessage(messages.componentsTab)} />
               <Tab eventKey={ContentType.collections} title={intl.formatMessage(messages.collectionsTab)} />
             </Tabs>
+            <ActionRow className="my-3">
+              <SearchKeywordsField />
```

Comment:
```suggestion
              <SearchKeywordsField className='mr-3' />
```

To fix spacing between search and filters.